content/contents+de.lr

```diff
@@ -82,98 +86,67 @@ resources_materials_heading: Materialien
 ---
 resources_materials_text:
 
-Erzeuge doch ein Logo mit diesem [Logo-Generator](http://odd.structure-and-representation.com/)!
+Erzeuge ein Logo für deiner Stadt mit diesem [Logo-Generator](http://odd.structure-and-representation.com/)!
```

Comment:
Erzeuge ein Logo für deine Stadt mit diesem [Logo-Generator](http://odd.structure-and-representation.com/)!